uniauth/views.py

```diff
@@ -147,29 +159,46 @@ def login(request):
         if num_institutions == 1:
             return HttpResponseRedirect(institutions[0][2] + query_params)
 
+        elif api_mode:
+            # return institutions and queryparams to be displayed on custom UI login form
+            return JsonResponse({'institutions': institutions, 'queryparams': query_params})
```

Comment:
❓ A bit confused by this too. Presumably, in the JWT flow, we wouldn't ever be displaying the UI form. Or are we returning the `institutions` and `queryparams` to facilitate client apps creating their own login forms?

Reply:
⛏️ Also, nitpick, but JSON keys should be camelCased.